apps/editors/models.py

```diff
@@ -17,6 +17,7 @@
 from amo.urlresolvers import reverse
 from amo.utils import cache_ns_key, send_mail
 from addons.models import Addon, Persona
+from constants.base import REVIEWED_OVERDUE_BONUS
```

Comment:
We usually don't explicitly import those, which are automatically imported as `amo.REVIEWED_OVERDUE_BONUS` (same for the tests below).